src/Spd.Resource.Repository/Biz/Mappings.cs

```diff
@@ -93,5 +96,29 @@ private static string SetAddressField(string src, string dest)
 
             return src;
         }
+
+        private static SwlContactInfo GetSoleProprietorContactInfo(IEnumerable<spd_licence> licences)
+        {
+            spd_licence? solePropietorInfo = licences.FirstOrDefault(l => l.statuscode == DynamicsConstants.StatusCode_Active);
+            
+            return new SwlContactInfo()
+            {
+                BizContactId = solePropietorInfo?.spd_LicenceHolder_contact?.spd_contact_spd_businesscontact.FirstOrDefault()?.spd_businesscontactid,

```

Comment:
BizContactId should be null, sole proprientor licence does not have bizContact

Reply:
If the field is always null, why do we need to include it as part of the response?

Reply:
no, in this situation, it is null.
The SwlContactInfo type will also be used in ControllingMember, Employee etc. At that scenario, it won't be null.

Reply:
Got it, fixed.